plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.tsx

```diff
@@ -209,14 +215,21 @@ type ScoreCardTableProps = {
   title?: string;
   entityKindFilter?: string[];
 };
-export const ScoreCardTable = ({title, entityKindFilter}: ScoreCardTableProps) => {
-  const { loading, error, value: data } = useScoringAllDataLoader(entityKindFilter);
+export const ScoreCardTable = ({
```

Comment:
To distinguish between the two "table" components let's rename this to ScoreBoardTable in some followup PR

Reply:
Sure!